test/bin/run-tests-docker.sh

```diff
@@ -4,34 +4,80 @@ set -e
 unset CDPATH
 cd "$( dirname "${BASH_SOURCE[0]}" )/../.."
 
-IMAGE_NAME_SCITRAN_CORE=scitran-core
+IMAGE_NAME_SCITRAN_CORE=${IMAGE_NAME_SCITRAN_CORE:-"scitran-core"}
 IMAGE_NAME_MONGO=mongo
-CONTAINER_NAME_MONGO=some-mongo
+CONTAINER_NAME_MONGO=scitran-core-test-mongo
+CONTAINER_NAME_SCITRAN_CORE=${CONTAINER_NAME_SCITRAN_CORE:-"scitran-core-test-uwsgi"}
```

Comment:
This container is ephemeral. Like above, I'd suggest not supporting the caller to override.